client/connection.go

```diff
@@ -35,6 +44,9 @@ type ConnectionArgs struct {
 
 	// Controls whether a client verifies the server's certificate chain and host name.
 	InsecureSkipVerify bool
+
+	// Persistent cookie jar
+	CookieJar *cookiejar.Jar
```

Comment:
Looks to me like this could be replaced by http.CookieJar, getting us rid of the persistent-cookiejar import from within the client/ package.

Reply:
good point, done